kernel-scala/src/main/scala/urth/widgets/util/SerializationSupport.scala

```diff
@@ -50,10 +50,13 @@ trait SerializationSupport extends LogLike {
   def dataFrameWrites(limit: Int) = Writes {
     (df: DataFrame) => {
       val columns: Array[String] = df.columns
-      val data: Array[Array[String]] =
+      val data: Array[Array[JsValue]] =
         df.take(limit).map(row => {row.toSeq.toArray.map {
```

Comment:
There are just more types that Json.toJson() supports that we are not considering in the switch. i.e Map, List, Array, Tuples, etc.